.github/pull-request-template.md

```diff
@@ -5,13 +5,41 @@
 4. Update the relevant documentation.
 -->
 
+**Decision log**
+<!-- Required for pull requests with the decision record only -->
```

Comment:
Instead of creating a different sections with such comments, would it be possible to create a second pull request template solely for decisions?

Reply:
not possible - I mean you can create a second template, but then it does not work like with issues, you need to put in the URL the parameter that will point to the specific template, what is not user friendly. So either we will have one generic template or we need to come back to the concept of having separate DR document

Reply:
PR template updated - left only the link to the DR template